src/main/java/ch/admin/bag/covidcode/authcodegeneration/web/controller/AuthCodeVerificationControllerV2.java

```diff
@@ -42,6 +43,21 @@ public AuthCodeVerificationControllerV2(AuthCodeVerificationService authCodeVeri
         return ResponseEntity.ok().body(accessTokenWrapper);
     }
 
+    @Operation(summary = "Get onset date for authorization code")
+    @PostMapping(value="/date")
+    public ResponseEntity<AuthorizationCodeOnsetResponseDto> getOnset(@Valid @RequestBody AuthorizationCodeVerificationDto verificationDto) {
+        var now = Instant.now().toEpochMilli();
+        log.debug("Call of getOnset with authCode '{}'.", verificationDto.getAuthorizationCode());
+        final AuthorizationCodeOnsetResponseDto onsetWrapper =
+            authCodeVerificationService.getOnsetForAuthCode(
+                verificationDto.getAuthorizationCode(), verificationDto.getFake());
+        if (onsetWrapper == null || onsetWrapper.getOnset() == null) {
+            throw new ResourceNotFoundException(null);
+        }
+        normalizeRequestTime(now);
```

Comment:
as in the "verify()" method it makes more sense to normailzeRequestTime() before throwing the ResourceNotFoundException